public/calendar_reboot.js

```diff
@@ -35,9 +35,12 @@ function CheckIt() {
     
     var monthObjects;
     
+    this.backgroundActivityChange = new CustomEvent('backgroundActivityChange');
```

Comment:
Hmm.. can you be more specific with "Usually the dispatcher just provides a function can call if you want to hook a listener up." How does it work? Or point me in the direction where I can read about it.